corehq/apps/users/views/mobile/users.py

```diff
@@ -671,6 +671,9 @@ def get_pagination_data(self, in_data):
 
         # backend pages start at 0
         users_query = self._user_query(query, page - 1, limit)
+
+        # run with a blank query to fetch total records with same scope as in search
+        total_records = self._user_query('', 0, 0).run().total
```

Comment:
nit: should use `count` here instead of `run().total`. it also ensures that `size` is `0` so it doesn't fetch any results